packages/v4/patternfly-docs/content/contribute/about/about.md

```diff
@@ -5,24 +5,20 @@ section: contribute
 
 The PatternFly open source community depends on contributions to help our design system grow and evolve. We encourage everyone, regardless of background, to make suggestions for enhancements, contribute new design patterns and ideas, help identify bugs in code, and more. With your help, we can stay on top of the latest and greatest implementation solutions.  
 
+## PatternFly feature lifecycle
```

Comment:
I wonder if this new section would make more sense below the "PatternFly on GitHub" section? Just because the graphic mentions "Core" and "patternfly-react", but I worry that readers wouldn't understand those terms until they read the following section.

Reply:
Makes sense. Will move it.